examples/LexActivator.pas

```diff
@@ -499,14 +603,133 @@ function GetLicenseAllowedActivations: LongWord;
 
 function GetLicenseTotalActivations: LongWord;
 
+(*
+    FUNCTION: GetLicenseTotalDeactivations()
+
+    PURPOSE: Gets the total deactivations of the license.
+
+    RESULT: Total deactivations of the license.
+
+    RETURN CODES: ELAFailException, ELAProductIdException, ELATimeException,
+    ELATimeModifiedException
+*)
+
+function GetLicenseTotalDeactivations: LongWord;
+
+(*
+    FUNCTION: GetLicenseAllowedDeactivations()
+
+    PURPOSE: Gets the allowed deactivations of the license.
+
+    RESULT: Allowed deactivations of the license.
+
+    EXCEPTIONS: ELAFailException, ELAProductIdException, ELATimeException,
+    ELATimeModifiedException
+*)
+
+function GetLicenseAllowedDeactivations: LongWord;
+
```

Comment:
@muneebkq `LongWord` in Delphi does not support -ve values as it represents `unsigned int`. Please make the changes to the `GetLicenseAllowedActivations()` as well. This is because we now support to -1 as a sentinal value.